src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml

```diff
@@ -48,6 +59,15 @@
                 </Button>
             </StackPanel>
             <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" Grid.Column="1">
+                <Button Content="{x:Static p:Resources.PythonScriptEditorMoreInfoButton}" 
+                        Style="{DynamicResource ResourceKey=STextButton}" 
+                        Name="MoreInfoButton" 
+                        Click="OnMoreInfoClicked" 
+                        HorizontalAlignment="Right" 
+                        VerticalAlignment="Bottom" 
+                        Margin="2" 
+                        Grid.Column="1"
+                        Visibility="{Binding nodeModel.Engine, Converter={StaticResource PythonEngineVersionToVisibilityConverter}}"/>
```

Comment:
@mmisol also any suggestion on debugging this? I found my converter not triggered at all

Reply:
Oh. I think the problem is not with your button, but that the engine combo is not firing a change. I think you had to do `UpdateSourceTrigger=PropertyChanged`.